vcd/resource_vcd_vapp_network.go

```diff
@@ -57,11 +57,22 @@ func resourceVcdVappNetwork() *schema.Resource {
 				Description: "Optional description for the network",
 			},
 			"netmask": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				ForceNew:    true,
-				Default:     "255.255.255.0",
-				Description: "Netmask address for a subnet. Default is 255.255.255.0",
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				Deprecated:   "Use prefix_length instead",
```

Comment:
Could be a good hint that `prefix_length` also supports IPv6
```suggestion
				Deprecated:   "Use prefix_length instead which supports both IPv4 and IPv6",
```

Reply:
Fixed